fs_mark/test_armv7a7-imx6ull-evk.yaml

```diff
@@ -0,0 +1,132 @@
+# This yaml contain armv7a7-imx6ull-evk specific fs_mark tests
+test:
+  type: harness
+  harness: fs_mark.py
```

Comment:
As we discussed I would consider adding some nested cases to provide better coverage.

Reply:
This tests are not straight forward to implement, maybe it is better to add it later on.